src/android/EntryActivity.java

```diff
@@ -1,4 +1,4 @@
-package __PACKAGE_NAME__.wxapi;
+package com.smartly.hybrid.wxapi;
```

Comment:
Hard code the package name since we don't need it in Miya Miya.

NOTE: this would be an issue we'd need to solve it, for instance, we decide to release a Valar mobile app. But we'll cross that bridge when we come to it.